conans/client/generators/visualstudio_multi.py

```diff
@@ -14,7 +14,8 @@ def __init__(self, settings):
 
         toolset = settings.get_safe("compiler.toolset")
         if not toolset:
-            default_toolset = {"15": "v141",
```

Comment:
@SSE4 Do you know if it is temporary and they will release the vs "16" with a new toolset? Is it maybe confirmed that the toolset will be `v141`? We need to look at it carefully, maybe our model is already broken.

Reply:
probably, I have something wrong with my VS 2019 installation. it updated to the latest, and doesn't have any `v142` toolset.
also, in installer, I don't see any options to install `v142` toolset.
however, there are multiple sources indicating it should exist, e.g.:
- https://cmake.org/cmake/help/git-stage/variable/MSVC_VERSION.html
- https://docs.microsoft.com/da-dk/cpp/porting/features-deprecated-in-visual-studio?view=vs-2017
this might be due to the fact I have multiple Visual Studio versions already installed.
another weird point, is that `v141` toolsets from VS 2017 and VS 2019 have different compiler versions in them, so I am not really fully sure about it.
I'll try to install it on clean Virtual Machine and let you know.

Reply:
I've installed Preview on clean machine, and there is only `v141` toolset available as well.

Reply:
checked CMake source code, and found:
https://github.com/Kitware/CMake/commit/57e48f16f2c3b73fb27057267c8092a41005bb75#diff-9893c1c749828e3bc2d729d697a316edR60

Reply:
I suppose we now have enough information to confirm `v142` should be default toolset. I'll update my PR accordingly.

Reply:
changed to v142